packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -1831,3 +1831,14 @@ export function cartesian<T>(matrix: T[][]): T[][] {
         [[]]
     )
 }
+
+export function isElementHidden(element: Element | null): boolean {
+    if (!element) return false
+    const computedStyle = window.getComputedStyle(element)
```

Comment:
Does `getComputedStyle` also take into account whether parent elements are hidden?

Reply:
![image](https://github.com/owid/owid-grapher/assets/11844404/486aac2f-9d3d-4c6c-b255-4d8e8a5dea75)

Nope, hence the recursive to the element's parent

Reply:
Oh dang, yeah I totally didn't see that call